deployment/packagebuild/packages.d/vnet/openvnet-vnmgr/post-install

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -e
+
+user="vnet-vnmgr"
+logfile="/var/log/openvnet/vnmgr.log"
+
+if ! id "$user" > /dev/null 2>&1 ; then
+    adduser --system --no-create-home --shell /bin/false "$user"
+fi
+
+touch "$logfile"
```

Comment:
the openvnet-common recipe creates the folder : https://github.com/axsh/openvnet/blob/master/deployment/packagebuild/packages.d/vnet/openvnet-common/recipe.rb#L44